.gitignore

```diff
@@ -4,6 +4,11 @@ assets/
 /Test Results - .html
 /geckodriver.log
 env
+./*.dmg
+./*.tar
+./*.zip
+./*.gz
+./*.7z
```

Comment:
Not sure if it matters, but these were the only instances of `./` prefix in the file.
The rest were either `/` prefix (like `/venv/` on L1 above) or no prefix (like `assets/` on L3 above, or `*.so` on L19 below).

Reply:
I'll change that